### PR TITLE
added new color scheme and styling to all components

### LINKED ISSUE #34: Begin Design Mockup
Use Tailwindcss to design the app according to the following wireframe and color scheme.

image

### LINKED ISSUE #35: Remove navigation from home screen
The Add an Item and Listview buttons are not necessary on the home screen.

### PR DESCRIPTION
**Pull request by shelleymcq**
## Description

Renamed our app "Smart Shopper".

Added our color scheme, button styling, and updated text to all components. We're using tailwind colors: sky-700 (#0369a1) for the blue background, teal-500 (#14b8a6) for the green, amber-300 (#fcd34d) for the yellow, and rose-500 (#f43f5e) for the pinkish red.

Removed Navigation buttons from the home screen.

## Acceptance Criteria

This is the first draft of our new design. See upcoming tickets for next steps.

## Related Issue
Closes #34
Closes #35

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|    | :sparkles: New feature     |
|  ✓  | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

image

### After

image

## Testing Steps / QA Criteria

The new styling and responsiveness can be testing by pulling the branch down to your local environment.

### CODE REVIEW

File: src/configuration.js [hunk 1]
```diff
@@ -63,5 +63,6 @@ export const navButtonStyles = {
   bottom: '0',
   width: '100%',
   textAlign: 'center',
-  backgroundColor: '#fff',
+  backgroundColor: '#0369a1',
```
Comment (rudidev08) [inline src/configuration.js:66]: Should this be moved to tailwind css? (happy to help on slack)

File: src/index.css [hunk 2]
```diff
@@ -1,6 +1,7 @@
 @import url('https://fonts.googleapis.com/css2?family=Exo:wght@200&family=Inconsolata:wght@500&family=Staatliches&display=swap&family=Karma:wght@500&display=swap');
 body {
   margin: 0;
+  background-color: #0369a1;
```
Comment (rudidev08) [inline src/index.css:4]: Knowing tailwind css, usually index.css is as empty as possible. Font import is ok, but for body, I'd see if you can do apply this background color directly on body in index.html

**Comment by rudidev08** [comment]
Amazing, especially how you successfully implemented tailwind css, not a trivial feat!
Found two places (index.css and nav bar style) that are also good candidates to move to tailwind css, and I'd recommend adding comments on more complex tailwind css lines. Example:
{/* centered on page, with programmatically set background color based on group */}
<section className="...
or
{/* displays todo items as cells with columns that depend on screen size. On mobile, it will be one column, with max 5 on largest size */}
<div className="...

File: src/configuration.js (hunk 1 above)
Comment (shelleymcq) [reply src/configuration.js:66]: Yes, there are actually several little things that need to be switched from css to tailwind. I focused on broad strokes for this round and will dig in to the old stuff next.

**Pull request merged**